routers/retraining_pipeline.py

```
@@ -0,0 +1,212 @@
+        if task_id:
+            from celery_worker import celery_app
+            result = celery_app.AsyncResult(task_id)
+            pipeline["task_id"] = task_id
+            pipeline["task_state"] = result.state
+            pipeline["task_info"] = (
+                result.info if isinstance(result.info, dict) else {}
+            )
+
+        return {"success": True, "pipeline": pipeline}
+
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)